### DISCUSSION TITLE
A paying gig in in networked-aframe.

### DISCUSSION THREAD

**Discussion by Therealjosephchrzempiec** [networked-aframe/networked-aframe#431] [General]
hello, I really need help with something in networked-aframe. I honestly can not do on my own. I'm wondering if anyone from the networked-aframe team or someone who works with networked-aframe is willing to do a side job?

Joseph

**Comment by vincentfretin** [comment]
Hello, if you need help for a few hours, we can schedule a pair programming session, see [my GitHub Sponsors page](https://github.com/sponsors/vincentfretin). Let's talk about it in Discord DM.

**Comment by Therealjosephchrzempiec** [comment]
Hello Vincent, I DM you in discord.

> Hello, if you need help for a few hours, we can schedule a pair programming session, see [my GitHub Sponsors page](https://github.com/sponsors/vincentfretin). Let's talk about it in Discord DM.
